Sources/Common/Store/SdkConfig.swift

```diff
@@ -144,6 +148,17 @@ public struct SdkConfig {
      */
     public var autoTrackScreenViews: Bool
 
+    #if canImport(UIKit)
+    /**
+     Filter automatic screenview events to remove events that are irrelevant to your app.
+
+     Return `true` from function if you would like the screenview event to be tracked.
```

Comment:
`     Return `true` from function if you would like the screenview event to be tracked.`

Is this specific to SwiftUI? Having `autoTrackScreenViews` and `filterAutoScreenViewEvents` seem to be a bit confusing to me.

Reply:
`autoTrackScreenViews` is to disable the feature entirely. 
`filterAutoScreenViewEvents` is to configure the feature to behavior that you want, assuming that `autoTrackScreenViews` is true. 

Therefore, both SDK config options provide a different value.